### PR TITLE
Fix stuff

### PR DESCRIPTION
**Pull request by Moosems**
Turns out, almost everything was working near perfect.

### CODE REVIEW

**Comment by Moosems** [comment]
Need to make it possible to hand a theme_provider to the linenumbers.

File: chlorophyll/codeview.py [hunk 1]
```diff
@@ -5,19 +5,19 @@
...
+import toml
 from pyperclip import copy
 from tklinenums import TkLineNumbers
 
 from .schemeparser import _parse_scheme
 
 color_schemes_dir = Path(__file__).parent / "colorschemes"
 
-LexerType = Union[Type[pygments.lexers.Lexer], pygments.lexers.Lexer]
+LexerType = Union[Type[pygments.lexer.Lexer], pygments.lexer.Lexer]
```
Comment (rdbende) [inline chlorophyll/codeview.py:20]: For this PR this is the change that matters. Please open a new PR for the linenumbers stuff.

Comment (Moosems) [reply chlorophyll/codeview.py:20]: So delete the others from this PR?

Comment (rdbende) [reply chlorophyll/codeview.py:20]: Yes, pls.

**Review by rdbende — APPROVED** [review]

**Pull request merged**